tox.ini

```diff
@@ -12,9 +12,9 @@ deps =
     beautifulsoup4==4.3.2
     mock==1.0.1
     pytest-cache==1.0
-    pytest==2.7.2
-    pytest-cov==1.8.1
-    pytest-django==2.8.0
+    pytest==2.8.3
+    pytest-cov==2.2.0
+    pytest-django==2.9.1
```

Comment:
I think these versions should be locked down. Would avoid needless checks against the PyPI registry, when we already have the correct wheels cached.